Milestone 2: Create works and aboutMe page

s2jack opened the pull request:
Create Works section
Use CssGrid and flexbox.
Fix padding-margin and color mistakes
Add font medias
Create About section
Organize text and boxes together
Create Social media icons and links

s2jack committed:
Add Fonts

Add all fonts used in website

s2jack committed:
Add images

Add all images used in portfolio website

s2jack committed:
Update index.html

Add about text and header
Add 3 projects by using Css Grid and Flexbox
Add social media icons and links
Add Technical Skills section by using Flexbox
Add buttons and links

s2jack committed:
Update styles.css

Add fonts
Use Css Grid and Flexbox
Create About and Works section

s2jack committed:
Update styles.css

Add Font medias
fix padding and orientation problems
Fix color and typo mistakes
Try to use CSS grid and flexbox together.

s2jack committed:
Update styles.css

Try to fix linters errors

s2jack committed:
Update styles.css

Try to fix minor psuedo error

abdulhamiid commented on index.html:
```diff
@@ -32,6 +32,94 @@ <h1 id="name" class="text">
+            <div class="tag box3">Javascript</div>
+          </div>
+          <div type="image" id="prdetailbutton"></div>
+        </div>
```
There are some issues with this section which we need to make an improvement.

- The project cards image is missing

- Your design
![image](https://user-images.githubusercontent.com/78251927/212252587-c0cc4df4-d1e3-4b4c-b8d3-2668e6c7d1d8.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/212252241-4e477e95-6cc2-4d31-bc22-924a5ac4a05a.png)

- The margin between a list of technologies `HTML`, `CSS`, and `JavaScript`
- The "See this project" button is missing, please add it. The button is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design)
- 
![image](https://user-images.githubusercontent.com/78251927/212256465-5ccb67c7-c138-4e68-ae70-868d30f023bd.png)


```html
<div class="projectname">Project name goes here</div>
```
![image](https://user-images.githubusercontent.com/78251927/212254594-3b01a793-645e-4dbe-b0a7-566cd68fecd5.png)

- The project title should use a header tag hx (examples: `<h2>`,`<h3>`, etc...). This way will be semantically correct and the screen readers and other technologies will thank you.


```html
      <div class="tags">
        <div class="tag box1">HTML</div>
        <div class="tag box2">CSS</div>
        <div class="tag box3">Javascript</div>
      </div>
```
- According to the project requirement please let's use ul and li for categories in the "Project" section. `Html`, `CSS`, and `Javascript` should be list items.

abdulhamiid commented on index.html:
```diff
@@ -32,6 +32,94 @@ <h1 id="name" class="text">
+            </div>
+      </div>
+    </section>
+    <h2 class="title">
+      About
+    </h2>
+    <section id="about">
+    <p id="bio">
```
Please let's adjust the top margin/padding of this section, according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), we must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your design 
![image](https://user-images.githubusercontent.com/78251927/212254957-59f7f938-a7b1-4bf3-b8dc-84d7a9a45cef.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/212255119-70e47756-080d-4eea-99dc-ed05dc050827.png)

abdulhamiid commented on index.html:
```diff
@@ -32,6 +32,94 @@ <h1 id="name" class="text">
+        <ul class="ulheader">Languages</ul>
+        <li class="article">Javascript</li>
+        <li class="article">HTML</li>
+        <li class="article">CSS</li>
```
Please kindly correct the mistake, the `<ul>` tag should be the wrapper for all the `<li>` tag.
```html
<ul>
  <li>. . . </li>
  <li>. . . </li>
  <li>. . . </li>
<ul>
```

abdulhamiid commented on index.html:
```diff
@@ -32,6 +32,94 @@ <h1 id="name" class="text">
+        <ul class="ulheader">Frameworks</ul>
+        <li class="article">NaN</li>
+        <li class="article">NaN</li>
+        <li class="article">Nan</li>
+        <img id="frameicon" src="images/ic_framework.png">
+      </div>
```
Please kindly use a background image for the decorations and not `img` tags, (decorations are always backgrounded images, not `img` tags)

![image](https://user-images.githubusercontent.com/78251927/212257000-30ce2b36-0126-4e3a-815a-c0a1b3d1e8c8.png)

![image](https://user-images.githubusercontent.com/78251927/212257339-9dbfc2ec-e837-4cb8-bc8f-7e660d6ecdc6.png)

s2jack replied:
@abdulhamiid hey, I hope you are well. First of all, I appreciate your time. I just want to ask about the difference between "img" and background images that you mentioned in the last article of your suggestions. What should I do exactly? Should I use the "background styled div" tag instead of "img" tag? I just want to make sure I understand the topic. Please let me know when you are available.

s2jack replied:
I already did the other changes. Please give some advice for the last one

GedeonTS replied:
Hi @s2jack , yes you have to assign to the div a `background-image:url(./image.png)`.

abdulhamiid reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @s2jack 👋 , 
![Alt Text](https://c.tenor.com/vOLW_UQMySIAAAAC/almost-there.gif)
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

s2jack committed:
Fix project images error

Fix the images source link in the index.html

s2jack committed:
Update styles.css

Set up margins between a list of technologies in project section

s2jack committed:
Add "h1" tag to Project titles

Add h1 tag to Project titles instead of div.

s2jack committed:
Fix project info section errors

Add "see project" button
Fix margins between all elements
Fix margins between technical skills
Fix Font sizes

s2jack committed:
Fix project info technical tags

Fix margins and sizes of the tags due to changed their tags from "div" to "ul - li"

s2jack committed:
Change technical tags to "ul-li"

Delete old wrapped elements in "div"
Create "ul-li" elements for technical tags in the project info section

s2jack committed:
Update styles.css

Fix linter error

s2jack committed:
Update index.html

Put li elements into ul elements in about section

s2jack committed:
Update styles.css

Fix ul and li elements orientations

s2jack committed:
Update styles.css

Fix linters error

s2jack committed:
Update styles.css

Fix margins between technical tags in the project info sections.

GedeonTS commented on index.html:
```diff
@@ -32,6 +32,106 @@ <h1 id="name" class="text">
+      nostrud exercitation ullamco laboris nisi<br>
+      ut aliquip ea eiusmod
+    </p>
+    <div type="image" id="cvbutton"></div>
+    <div id="techskills">
+      <div class="techbox" id="languages">
+        <h3 class="ulheader">Languages</h3>
+        <ul>
+          <li class="article">Javascript</li>
+          <li class="article">HTML</li>
+          <li class="article">CSS</li>
+        </ul>
+        <img id="langicon" src="images/ic_languages_mob.png">
+      </div>
+      <div class="techbox" id="frameworks">
+        <h3 class="ulheader">Frameworks</h3>
+        <ul>
+          <li class="article">NaN</li>
+          <li class="article">NaN</li>
+         <li class="article">Nan</li>
+        </ul>
+        <img id="frameicon" src="images/ic_framework.png">
+      </div>
+      <div class="techbox" id="skills">
+        <h3 class="ulheader">Skills</h3>
+        <ul>
+          <li class="article">Git</li>
+          <li class="article">GitHub</li>
+          <li class="article">Figma</li>
+        </ul>
+        <img id="skillicon" src="images/ic_skills.png">
+      </div>
```
- [ ] As requested by the previous reviewer [here](https://github.com/s2jack/create-portfolio-website/pull/4#discussion_r1068998648), kindly make sure to add the decoration images as background-image instead of div tags.

GedeonTS commented on index.html:
```diff
@@ -32,6 +32,106 @@ <h1 id="name" class="text">
+    </section>
+    <section id="contact">
+      <div id="contactbg">
+        <div id="socialicons">
+          <a href="https://stackoverflow.com"><img src="images/socialmedia-icons/ic_stackoverflow.png" alt="stackoverflow-icon"></a>
+          <a href="https://wwww.github.com/s2jack"><img src="images/socialmedia-icons/ic_github.png" alt="github-icon"></a>
+          <a href="https://twitter.com/1stGuler"><img src="images/socialmedia-icons/ic_twitter.png" alt="twitter-icon"></a>
+          <a href="https://www.linkedin.com/in/selcuk-guler/"><img src="images/socialmedia-icons/ic_linkedin.png" alt="linkedin-icon"></a>
+          <a href="https://medium.com/"><img src="images/socialmedia-icons/ic_medium.png" alt="medium-icon"></a>
+          <a href="https://www.angellist.com/"><img src="images/socialmedia-icons/ic_angellist.png" alt="angellist-icon"></a>
```
- [ OPTIOANAL] Kindly make sure the social icon links are wrapperd in an unordered list to respect the semantic HTML rules. :recycle:

GedeonTS reviewed (changes requested):
# STATUS: CHANGES REQUIRED :recycle: :
Hi @s2jack , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

<img src="https://media.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif">

 ## Highlights :tophat: :+1: 
- No linter errors :+1: 
- Good use of Github flow :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

s2jack committed:
Add files via upload

Add background images for technical skills section.

s2jack committed:
Update index.html

Delete img tags of technical skills section
Implement images into div

s2jack committed:
Update styles.css

Change technical skills attributes
Declare background images all parts of this section

s2jack committed:
Update styles.css

Fix linter errors

s2jack committed:
Update styles.css

Fix url errors

jerryowusu reviewed (approved):
Hi @s2jack  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![CookedDelectableDolphin-size_restricted](https://user-images.githubusercontent.com/65233860/212389968-2766895a-2947-4d81-bc4c-053085662578.gif)


## Highlights
All linters are passing :heavy_check_mark: 
Great effort in making the requested changes :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

s2jack commented:
@jerryowusu @abdulhamiid @GedeonTS Thank you all for everything during this process. I appreciate it. All of your predictions pushed me to be more paced and passionate.

Nyame-Wolf reviewed (commented):
# STATUS: INVALID CODE REVIEW :x: :stop_sign:  

Hi @s2jack , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :warning:

You have submitted a project, that is not for this particular milestone. Kindly read the lesson,`How to perform a code review` then follow the instructions for this particular exercise.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Make it work and then make it better!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
